tests/cupy_tests/core_tests/test_ndarray_reduction.py

```diff
@@ -20,6 +20,12 @@ def test_max_axis_large(self, xp, dtype):
         a = testing.shaped_random((3, 1000), xp, dtype)
         return a.max(axis=0)
 
+    @testing.for_dtypes(['?', 'b', 'h', 'e'])
+    @testing.numpy_cupy_allclose()
+    def test_max_axis_huge(self, xp, dtype):
+        a = testing.shaped_random((2, 1024, 1024), xp, dtype)
```

Comment:
This array seems not huge. Did you actually intend to write `(2048, 1024, 1024)` instead? If so, we will need `@slow` decorator here.